spec/models/product_spec.rb

```diff
@@ -5,4 +5,8 @@
     it { is_expected.to validate_presence_of(:name) }
     it { is_expected.to validate_presence_of(:description) }
   end
+
+  context 'relationships' do
+    it { is_expected.to have_many(:lots) }
```

Comment:
I know you love your is_expected to, I just have to express my love for SHOULDa matchers 😉